databricks/koala/tests/test_dataframe.py

```diff
@@ -217,6 +217,23 @@ def test_to_datetime(self):
         self.assert_eq(pd.to_datetime(s, infer_datetime_format=True),
                        pyspark.to_datetime(ds, infer_datetime_format=True))
 
+    def test_sort_values(self):
+        df = pd.DataFrame({'a': [1, 2, 3, 4, 5, 6, 7],
+                           'b': [7, 6, 5, 4, 3, 2, 1]})
+        ddf = self.spark.from_pandas(df)
+
+        ddf2 = ddf.sort_values('b')
+        df2 = df.sort_values('b')
+        self.assert_eq(ddf2, df2)
+
+        ddf2 = ddf.sort_values(['b', 'a'])
+        df2 = df.sort_values(['b', 'a'])
+        self.assert_eq(ddf2, df2)
+
+        ddf2 = ddf.sort_values(['b', 'a'], ascending=[False, True])
+        df2 = df.sort_values(['b', 'a'], ascending=[False, True])
+        self.assert_eq(ddf2, df2)
```

Comment:
What if the size of `by` and `ascending` are different?
Could you add the tests and follow the pandas behavior?